Application/EdFi.Admin.DataAccess/Utils/DatabaseNameBuilder.cs

```diff
@@ -2,73 +2,37 @@
 // Licensed to the Ed-Fi Alliance under one or more agreements.
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
- 
-using System.Configuration;
-using System.Data.SqlClient;
 
 namespace EdFi.Admin.DataAccess.Utils
 {
-    public class DatabaseNameBuilder
+    public static class DatabaseNameBuilder
     {
+        private const string EdFi = "EdFi_";
         private const string TemplatePrefix = "Ods_";
         private const string SandboxPrefix = TemplatePrefix + "Sandbox_";
 
-        private const string TemplateEmptyDatabase = TemplatePrefix + "Empty_Template";
-        private const string TemplateMinimalDatabase = TemplatePrefix + "Minimal_Template";
-        private const string TemplateSampleDatabase = TemplatePrefix + "Populated_Template";
-        public const string CodeGenDatabase = "EdFi_Ods";
-
-        private static string _databaseNameTemplate;
-
-        private static string DatabaseNameTemplate
-        {
-            get
-            {
-                return _databaseNameTemplate ??
-                       (_databaseNameTemplate =
-                           ConfigurationManager.ConnectionStrings["EdFi_Ods"] != null
-                               ? new SqlConnectionStringBuilder(
-                                   ConfigurationManager.ConnectionStrings["EdFi_Ods"]
-                                                       .ConnectionString).InitialCatalog
-                               : string.Empty);
-            }
-        }
+        public const string TemplateEmptyDatabase = TemplatePrefix + "Empty_Template";
+        public const string TemplateMinimalDatabase = TemplatePrefix + "Minimal_Template";
+        public const string TemplateSampleDatabase = TemplatePrefix + "Populated_Template";
+        public const string DemoSandboxDatabase = "EdFi_Ods";
 
         public static string EmptyDatabase
         {
-            get { return DatabaseName(TemplateEmptyDatabase); }
+            get => EdFi + TemplateEmptyDatabase;
         }
 
         public static string MinimalDatabase
         {
-            get { return DatabaseName(TemplateMinimalDatabase); }
+            get => EdFi + TemplateMinimalDatabase;
         }
 
         public static string SampleDatabase
         {
-            get { return DatabaseName(TemplateSampleDatabase); }
-        }
-
-        private static string DatabaseName(string databaseName)
-        {
-            return DatabaseNameTemplate.Contains("{0}")
```

Comment:
doesn't this remove the configurability of the templated connection string?
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/03a24fe2ecc8fbab411efb8656395e4995781294/Application/EdFi.Ods.SandboxAdmin.Web/Web.Npgsql.config#L14-L16

Reply:
This was never used, that is why i removed it.

Reply:
What do you mean? The `DatabaseNameTemplate` pulls the EdFi_Ods connection string from the web.config and  formats it with the populated template names. Our deployment sets it to `Database=EdFi_EFAStaging#{Octopus.Release.Channel.Name}_{0};` which allows it to differentiate sandboxes between different api versions.

Reply:
done

Reply:
done